lib/trellis/plugins/filter/filters.py

```diff
@@ -4,7 +4,12 @@
 from ansible.module_utils.six import string_types
 
 def to_env(dict_value):
-    envs = ["{0}='{1}'".format(key.upper(), str(value).replace("'","\\'")) for key, value in sorted(dict_value.items())]
+    envs = []
+    for key, value in sorted(dict_value.items()):
+        if isinstance(value, string_types) and '\n' in value:
+            envs.append('{0}="{1}"'.format(key.upper(), value.replace('"', '\\"')))
+        else:
+            envs.append("{0}='{1}'".format(key.upper(), str(value).replace("'", "\\'")))
```

Comment:
```suggestion
envs = ['{0}="{1}"'.format(key.upper(), str(value).replace('"','\\"\)) for key, value in sorted(dict_value.items())]
```

What if we just switch to double quotes for them all?

Reply:
Seems fine to me, simpler